# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rhythmbox/dbus-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur rhythmbox-0.9.5/metadata/rb-metadata-dbus-client.c rhythmbox-0.9.5-t2/metadata/rb-metadata-dbus-client.c
--- rhythmbox-0.9.5/metadata/rb-metadata-dbus-client.c	2006-05-27 06:15:44.000000000 +0200
+++ rhythmbox-0.9.5-t2/metadata/rb-metadata-dbus-client.c	2006-09-02 20:03:37.000000000 +0200
@@ -118,7 +118,7 @@
 	if (dbus_connection) {
 		if (dbus_connection_get_is_connected (dbus_connection)) {
 			rb_debug ("closing dbus connection");
-			dbus_connection_disconnect (dbus_connection);
+			dbus_connection_close (dbus_connection);
 		} else {
 			rb_debug ("dbus connection already closed");
 		}
diff -ur rhythmbox-0.9.5/metadata/rb-metadata-dbus-service.c rhythmbox-0.9.5-t2/metadata/rb-metadata-dbus-service.c
--- rhythmbox-0.9.5/metadata/rb-metadata-dbus-service.c	2006-05-27 06:15:44.000000000 +0200
+++ rhythmbox-0.9.5-t2/metadata/rb-metadata-dbus-service.c	2006-09-02 20:03:37.000000000 +0200
@@ -494,7 +494,7 @@
 	g_main_loop_run (svc.loop);
 
 	if (svc.connection) {
-		dbus_connection_disconnect (svc.connection);
+		dbus_connection_close (svc.connection);
 		dbus_connection_unref (svc.connection);
 	}
 
